src/fit_dataframe.jl

```diff
@@ -269,16 +268,12 @@ function expand_categoricals!(df::DataFrame,categoricals::Array)
     return expand_categoricals!(df, categoricalidxs)
 end
 
-# convert NaNs to NAs
-# isnan(x::NAtype) = false
-isnan(x::AbstractString) = false
-isnan(x::Union{T, Nothing}) where T = isnan(x.value)
```

Comment:
This was a bad find and replace from a previous PR that replaced `Nullable` with `Union{Nothing, T}`
It was always nonsense.
idk why it wasn't breaking tests earlier